packages/site/cms-site/src/cookies/useLocalStorageCookieApi.ts

```diff
@@ -1,20 +1,31 @@
 import React from "react";
 import { useLocalStorage } from "usehooks-ts";
 
-import { CookieApiHook } from "./CookieApiHook";
+import { CookieApi } from "./CookieApi";
 
 const localStorageCookieApiKey = "comet-dev-cookie-api-consented-cookies";
 
+declare global {
+    interface Window {
+        cometLocalStorageCookieApi: ReturnType<CookieApi>;
```

Comment:
```suggestion
        cometLocalStorageCookieApi: CookieApi;
```

I don't think that ReturnType is necessary here. ReturnType is for functions: https://www.typescriptlang.org/docs/handbook/utility-types.html#returntypetype

Reply:
`CookieApi` is the hook, that returns the list of consented cookies `consentedCookies` and the function that opens the cookie settings `openCookieSettings`. This window variable is to allow access to those directly, without having to call the hook.